knnvector/test_procedures/default.json

```diff
@@ -0,0 +1,74 @@
+{
+    "name": "no-train-test",
+    "default": true,
+    "schedule": [
+        {
+            "operation": {
+                "name": "delete-target-index",
+                "operation-type": "delete-index",
+                "only-if-exists": true,
+                "index": "{{ target_index_name | default('target_index') }}"
+            }
+        },
+        {
+            "operation": {
+                "name": "create-target-index",
+                "operation-type": "create-index",
+                "index": "{{ target_index_name | default('target_index') }}"
+            }
+        },
+        {
+            "name": "wait-for-cluster-to-be-green",
+            "operation": "cluster-health",
+            "request-params": {
+                "wait_for_status": "{{ cluster_health | default('green') }}"
+            }
+        },
+        {
+            "operation": {
+                "name": "custom-vector-bulk",
+                "operation-type": "custom-vector-bulk",
+                "param-source": "bulk-from-data-set",
+                "index": "{{ target_index_name | default('target_index') }}",
+                "field": "{{ target_field_name | default('target_field') }}",
+                "bulk_size": {{ target_index_bulk_size | default(500)}},
+                "data_set_format": "{{ target_index_bulk_index_data_set_format | default('hdf5') }}",
+                "data_set_path": "{{ target_index_bulk_index_data_set_path  }}",
+                "num_vectors": {{ target_index_num_vectors | default(1000) }}
+            },
+            "clients": {{ target_index_bulk_index_clients | default(1)}}
+        },
+        {
+            "operation": {
+                "name": "refresh-target-index",
+                "operation-type": "custom-refresh",
+                "index": "{{ target_index_name | default('target_index') }}",
+                "include-in-results_publishing": false,
+                "retries": 100
+            }
+        },
+        {
+            "name" : "force-merge-segments",
+            "operation" : "force-merge"
+        },
+        {
+            "name" : "warmup-queries-using-random-values",
+            "operation" : "warmup-queries",
+            "clients": {{ query_clients | default(10) }},
+            "iterations": {{ warmup_queries | default(100) }}
```

Comment:
this will not necessary load all the graphs in memory. The graphs from replicas may not be loaded.

Reply:
Got it. Should i change it to warmup API then?

Reply:
I would like to have both.

Reply:
Yes we should remove this and move this to warmup api call.

Also can we make the warmup step optional, where by default it is true